src/Tabs.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { memo } from 'react';
+import classNames from 'classnames';
+
+interface Props {
+  tabs: Tab[],
+  selectedTabId: string,
+  onSelect: (tab: Tab) => void,
+}
+
+export const Tabs: React.FC<Props> = memo(({ tabs, selectedTabId, onSelect }) => {
+  return (
+    <>
+      <div className="tabs">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              key={tab.id}
+            >
+              <button
+                className={classNames('button', { 'is-info': selectedTabId === tab.id })}
+                type="button"
+                onClick={() => onSelect(tab)}
```

Comment:
You could have put the entire callback in a separate one applying `useCallback` so that re-rendering doesn't cause create a new handler

Reply:
I didn't undestand this comment. Can you suggest please? What is callback might be look like?

Reply:
```
const handleButtonClick = useCallback(() => {
      if (!isSelected) {
        onTabSelected(tab);
      }
    }, [selectedTabId]);

    return (
      <>
        <button
          className="Tab__title"
          type="button"
          disabled={isSelected}
          onClick={handleButtonClick}
```

Reply:
All that is a just a matter of optimization. However, here is a recipe - instead of creating a callback inside `onClick` explicitly, you create a handler using `useCallback`

Reply:
NB: If you see, `isSelected` doesn't matter here, that's for UI